wen-restart/src/wen_restart.rs

```diff
@@ -690,9 +691,22 @@ pub(crate) fn aggregate_restart_heaviest_fork(
                 total_stake
             );
             let can_exit = total_active_stake_seen_supermajority >= majority_stake_required;
-            // Only send out updates every 30 minutes or when we can exit.
+            let saw_supermajority_first_time =
+                if current_total_active_stake >= majority_stake_required {
+                    if !total_active_stake_higher_than_supermajority {
+                        total_active_stake_higher_than_supermajority = true;
+                        true
+                    } else {
+                        false
+                    }
+                } else {
+                    false
+                };
```

Comment:
I think this can be simplified to 

```
let saw_supermajority_first_time = 
    current_total_active_stake >= majority_stake_required && 
    !total_active_stake_higher_than_supermajority && {
        total_active_stake_higher_than_supermajority = true;
        true
    };
```

Reply:
Done.